intuita-webview/src/codemodList/TreeView/index2.tsx

```diff
@@ -0,0 +1,235 @@
+import { useCallback, useState } from 'react';
+import { RunCodemodsCommand, CodemodHash } from '../../shared/types';
+import { vscode } from '../../shared/utilities/vscode';
+
+import { useProgressBar } from '../useProgressBar';
+
+import {
+	CodemodNode,
+	CodemodNodeHashDigest,
+	CodemodTree,
+} from '../../../../src/selectors/selectCodemodTree';
+import { IntuitaTreeView } from '../../intuitaTreeView';
+import { getCodemodNodeRenderer } from '../codemodNodeRenderer';
+import Popover from '../../shared/Popover';
+import { VSCodeButton } from '@vscode/webview-ui-toolkit/react';
+
+import s from './style.module.css';
+
+type Props = Readonly<{
+	tree: CodemodTree;
+	autocompleteItems: string[];
+	searchQuery: string;
```

Comment:
IMHO we should call it `searchPhrase` to have the same names for the same concept everywhere (and search query is mostly associated with URLs I think)